src/dummy.js

```diff
@@ -17,10 +17,11 @@ const media = [
     thumb: 'https://s3-us-west-1.amazonaws.com/powr/defaults/image-slider2.jpg',
     src: 'https://s3-us-west-1.amazonaws.com/powr/defaults/image-slider2.jpg',
     caption: '<h4>Elephant</h4>',
+    type: 'image/jpg'
```

Comment:
```suggestion
    type: 'image'
```

Is `jpg` really need?

Reply:
No it's not, this is not mime type, type is used by the lightbox to determine how to show the media.